apollo/src/graphql/model/model.ts

```diff
@@ -123,5 +126,13 @@ export class ObjectionMLModel extends Model {
                 to: 'dstkUser.user.userId',
             },
         },
+        cursor: {
```

Comment:
Are these new relations ever used? Glancing through stuff looks like they're safe to leave off.

I think we'll probably down the road also want a generic `cursors` table like you mentioned, so may be better to not have a reliance on these joins just in anticipation of rhat

Reply:
Yeah they aren't been used for anything right now. I put them is as a "just in case" but I'll remove and modify the patch so that it's agnostic to whatever table we link it to.